org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/model/ManifestAccess.xtend

```diff
@@ -38,6 +38,8 @@ class ManifestAccess extends TextFileAccess implements IGuiceAwareGeneratorCompo
 	
 	TypeReference activator
 	
+	String lineDelimiter = "\r\n"
```

Comment:
Not really. It is just the default value, which is overwritten by setLineDelimiter (available through @Accessors). But the default should be Strings.newLine(). I'll change that.